setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Unfortunately no. The file dialog isn't the problem in this case, it's the lack of a folder dialog. The code sample in that link is for enabling multiple select in file dialogs (also it's using win forms, but it's basically the same in WPF anyway), although it does link to examples for building your own custom folder dialog. 

The point I was trying to make in my previous comment was that I'd prefer not to build a custom one, although I will if insisted, since the result will be significantly worse than the native version exposed by the dependency. It would also add a non-trivial amount of additional code to the project that would need to be maintained should anybody ever decide to expand on the editor.